http-client-shared-data/src/main/java/org/triplea/http/client/JsonEncoder.java

```diff
@@ -0,0 +1,27 @@
+package org.triplea.http.client;
+
+import java.lang.reflect.Type;
+
+import feign.RequestTemplate;
+import feign.codec.Encoder;
+import feign.gson.GsonEncoder;
+
+/**
+ * A custom encoder that will be a pass-thru to Jackson encoder when encoding JSON objects,
+ * and a pass-thru of simple String values. Without this, String values are encoded to have surrounding
+ * quotes, this custom encoder will fix that so that Strings are passed as simple string values without
+ * extra surrounding quotes.
+ */
+public class JsonEncoder implements Encoder {
+  private static final GsonEncoder gsonEncoder = new GsonEncoder();
+
+  @SuppressWarnings("deprecation")
```

Comment:
We already had this in the other PR, although I forgot the exact details, but wouldn't this already be a follow-up PR?
Fixing this should be rather straight forward, a simple wrapper object is required.

Regardless I wouldn't suppress the warning if the fix is so simple.

Reply:
My 2 cents, a follow-up PR would have that as a dedicated change so that we're not debugging multiple things at the same time. I'm also not sure if/when I'll get to this, using the deprecated API here does not bother me, I don't see the real benefit so long as we are on the current version of feign. In principle I do support fixing this and would/will if I can get to this level of priority. I'm pushing to get all the testing written.